### PR TITLE
Add parts to video and audio, run all builds on dev

### LINKED ISSUE #125: Bug: No way to style `<video>` element inside `<mux-video>`
### Which Mux Elements/Packages has a bug?

- mux-video

### Which browser(s) are you using?

- Chrome

### Which operating system(s) are you using?

Mac

### How are you using Mux Elements?

`<script async src="https://unpkg.com/@mux-elements/mux-video"></script>`.

### Describe the bug

I want to style the`<video>` element inside of the `<mux-video>` element. In particular, I'd like to add `object-fit: cover`.

Pasted_Image_3_8_22__2_30_PM

## The current behavior

As far as I can tell, there is no way to style this element.

## The expected behavior

I want to be able to style it. A simple solution would be to put a `part="video"` attribute on the `<video>` element. That way one could style it as described in https://developer.mozilla.org/en-US/docs/Web/CSS/::part. I've tested this out by adding the `part` attribute in Chrome Devtools and it works.

image

### PR DESCRIPTION
**Pull request by gkatsev**
The part attributes for `mux-video` and `mux-audio` fix #125.

### CODE REVIEW

**Review by cjpillsbury — APPROVED** [review]
😎

File: packages/mux-video/src/CustomVideoElement.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ template.innerHTML = `
 
 </style>
 
-<video crossorigin></video>
+<video part="video" crossorigin></video>
```
Comment (luwes) [inline packages/mux-video/src/CustomVideoElement.js:36]: awesome 👍

**Review by luwes — APPROVED** [review]
🚢

**Comment by balexand** [comment]
Thanks!

**Pull request merged**